src/website/views/javascript/drawer.pug

```diff
@@ -144,7 +144,31 @@ section.a-table.a-table__options.-bordered.-my--3
               | Lets the developer referer the target programmatically instead of by data-target attribute.
               | It accepts Element object and xpath selector string.
 
-h4 Preventing memory leaks
+h3.mt--2 Methods
+.p.-text.-mb--3
+  .p-text There are three 
```

Comment:
Please use markup used in https://github.com/dani-cl-madrid/ux-chi/blob/ab292cf61329270f2472cd1862603931b53eb0ca/src/website/layouts/partials/stencilDoc.pug